constants/Enchants.json

```diff
@@ -501,19 +495,19 @@
     "smoldering": {
       "loreName": "Smoldering",
       "nbtName": "smoldering",
-      "goodLevel": 1,
+      "goodLevel": 0,
       "maxLevel": 5
     },
     "strong mana": {
       "loreName": "Strong Mana",
       "nbtName": "strong_mana",
-      "goodLevel": 7,
+      "goodLevel": 0,
       "maxLevel": 10
     },
     "tabasco": {
       "loreName": "Tabasco",
       "nbtName": "tabasco",
-      "goodLevel": 2,
+      "goodLevel": 0,
```

Comment:
when there are only level 1 2 3 for tabasco, i like the idea of having three different colors.